db/db.go

```diff
@@ -1181,6 +1181,13 @@ func GetTotalValidatorsCount() (uint64, error) {
 	return totalCount, err
 }
 
+// GetActiveValidatorCount will return the total-validator-count
+func GetActiveValidatorCount(currentEpoch uint64) (uint64, error) {
+	var count uint64
+	err := DB.Get(&count, "SELECT COUNT(*) FROM validators WHERE activationepoch >= $1 AND $2 < exitepoch", currentEpoch, currentEpoch)
```

Comment:
Needs an db index in order to be performant, otherwise the query will cause a full table scan